root/elections/ElectionVoting.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const Frag = require('../components/Frag');
+const {l, lp} = require('../static/scripts/common/i18n');
+const EditorLink = require('../static/scripts/common/components/EditorLink');
+const formatUserDate = require('../utility/formatUserDate');
+import {canCancel, canSecond, canVote, isInvolved, votesVisible} from '../utility/voting';
+const ExpirationDate = require('../components/ExpirationDate');
+
+type PropsT = {
+  +election: AutoEditorElectionT,
+  +user?: EditorT,
+};
+
+const ElectionVoting = ({election, user}: PropsT) => {
+  let message = l('To find out if you can vote for this candidate, please {url|log in}.',
+    {__react: true, url: '/login'});
+  if (user) {
+    if (election.is_closed) {
+      message = l('Voting is closed.');
+    } else if (election.is_pending) {
+      message = l('Voting is not yet open. If you would like to support this candidate, you can second their nomination. If you do not support this candidate, please note that you cannot cast a "No" vote (or abstain) until two seconders have been found.');
```

Comment:
You changed the order of the elsifs here compared to the original template, so now it will show the user this message (suggesting they can second the nomination) even if they aren't an auto-editor.